paper.md

```diff
@@ -0,0 +1,111 @@
+    orcid: 0000-0001-9226-3104
+    affiliation: 1 # (Multiple affiliations must be quoted)
+  - name: Steven Hollingsworth
+    corresponding: true # (This is how to denote the corresponding author)
+    affiliation: 2
+  - name: Day Fey
+    affiliation: 2
+  - name: Mary C Norris
+    affiliation: 2
+  - name: Walter Yu
+    affiliation: 3
+  - name: Kristiina Kerge
+    affiliation: 4
+  - name: Kris Haamer
+    affiliation: 4
+  - name: Gina 
+    affiliation: 2
+  - name: Brianda
+    affiliation: 2
+affiliations:
+ - name: Moore Institute for Plastic Pollution Research, USA
+   index: 1
+ - name: Code for Sacramento and Fresno, USA
+   index: 2
+ - name: California Department of Transportation, USA
+   index: 3
+ - name: Let's Do It Foundation, Estonia
+   index: 4
+date: 1 September 2022
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Although computer vision classification routines have been created for trash, they have not been accessible to most researchers due to the challenges in deploying the models. Trash AI is a web GUI (Graphical User Interface) for serverless computer vision classification of batch images with trash in them hosted at www.trashai.org. With a single batch upload and download, a user can automatically describe the types and quantities of trash in all of their images. 
+
+# Statement of need
+
+The trash in the environment is a widespread problem that is difficult to measure. Classical measurement techniques require surveyors with pen and paper to manually quantify every piece of trash at a site. This method is time-consuming. Scientists are actively trying to address this issue by using imaging to better understand the prevalence and distribution of trash in an `efficient yet effective manner` [@Majchrowska:2022; @Proença:2020; @Moore:2020; @vanLieshout:2020; @WADEAI:2020; @Lynch:2018; @Wuu:2018; @Waterboards:2018]. An app-based reporting of trash using cell phones, laptops, and other devices has been a `valuable solution` [@Lynch:2018]. Applications for AI in detecting trash currently include: images from `bridges` [@vanLieshout:2020], `drone imaging` [@Moore:2020], cameras on `street sweepers` [@Waterboards:2018], and cell phone app based reporting of `trash` [@Lynch:2018]. Although there are many artificial intelligence algorithms developed for trash classification, none are readily accessible to the average data scientist. The primary limitation is that artificial intelligence (AI) algorithms are primarily run through programming languages (not graphic user interfaces), difficult to deploy without AI expertise, and often live on a server (which costs money to host). New developments in browser-side AI (e.g. tensorflow.js) and serverless architecture (e.g. AWS Lambda) have created the opportunity to have browser-side artificial intelligence in a web GUI alleviating both obstacles. We present Trash AI, an open source service for making computer vision available to anyone with a web browser and images of trash. 
```

Comment:
> average data scientist

should this be `littler researcher`?  I feel like data scientists would be the ones who know how to run models.

> and serverless architecture (e.g. AWS Lambda)

Currently lambda isn't really doing anything special or enabling any workflows that wouldn't be possible with other app hosting methods.  We are primarily using it right now because it is currently the cheapest way to host our simple back-end application (most of the logic is on the front-end).

Reply:
Updated to litter researcher, good idea. 

Added the word affordable for the second part, so that it is clear that is why we are going serverless.

Reply:
https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d